tests/test_cli.py

```diff
@@ -7,6 +8,37 @@
 import numpy as np
 import sh
 
+from microscopium import io
+
+
+def assert_close(current, expected):
+    np.testing.assert_allclose(current, expected, atol=1e-3, rtol=1e-3)
+
+
+def check_reference_feature_json(output, reference_file):
+    """Compare JSON to a reference using knowledge about its contents.
+
+    Parameters
+    ----------
+    output : iterable of string
+        The output being tested. Each string must contain valid JSON.
+    reference_file : iterable of string
+        The reference against which the output is being compared.
+    """
+    for line, reference in zip(output, reference_file):
+        if not line and reference == '\n':
+            continue  # ignore blank lines
+        d = json.loads(line)
+        dref = json.loads(reference)
+        if 'feature_vector' in d:
+            assert_close(d['feature_vector'], dref['feature_vector'])
+        elif 'pca_vector' in d:
+            assert_close(d['feature_vector_std'], dref['feature_vector_std'])
```

Comment:
No, but nice call, I'll add a comment. What's happening is that both the PCA and the feature-vector std are emitted in the same json line.